Mesh4all/m4a-firmware
# dist/tools: Fixed names to m4a template generator

### Contribution description
Was changed the directory's name from `m4a-generator` to `template_generator`, also was fixed the source path of template_generator in the  script `m4a_installer.sh` .



### Testing procedure
You should has uninstalled pip and m4agen.
if previously has installed these tools
```sh
sudo pip uninstall m4agen
sudo apt remove pyhton3-pip
```
then run in the base directory:
```
make generate-example
```
Automatically, the makefile will offer you install pip and m4agen. Pip is offered as a dependency to run m4agen



### Issues/PRs references
Fixes #334
